action.yaml

```
@@ -161,6 +161,7 @@ runs:
         INPUT_UPLOAD_LANDING_STATE=$(payload uploadLandingState)
         INPUT_UPLOAD_SERIES=$(payload uploadSeries)
         INPUT_UPLOAD_ID=$(payload uploadId)
+        INPUT_AUTOFIX=$(payload autofix)
```

Let's make this naming a bit more obvious

```suggestion
        INPUT_AUTOFIX_AND_PUSH=$(payload autofixAndPush)
```